spec/controllers/medicaid/income_job_number_controller_spec.rb

```diff
@@ -1,13 +1,33 @@
 require "rails_helper"
 
-RSpec.describe Medicaid::IncomeJobNumberController, type: :controller do
+RSpec.describe Medicaid::IncomeJobNumberController do
+  let(:step) { assigns(:step) }
```

Comment:
Is there a down side to moving this directly in the context of the test in line 15? Since it's only used in one test I think it should be able to be housed in just that one test

Reply:
I was getting a `nil` error when calling a method on on `step` when instantiating this in the spec itself (that's what I started with)

putting it in a `let` did some kind of magic that fixed that. Do you know where in a controller spec is ideal to assign an `assigns` ?

Reply:
Done at e97f99c

Turns out you need to assign `assigns` after the `get` in the spec